data/README.md

```diff
@@ -0,0 +1,85 @@
+
+The images used for training this model were sourced from the following collections:
+1. [Heliconius Collection (Cambridge Butterfly)](#heliconius-collection-cambridge-butterfly)
+2. [Monteiro Dataset](#monteiro-data)
+3. [Smithsonian Tropical Research Institute (STRI) Data](#smithsonian-tropical-research-institute-stri-data)
+
+## [Heliconius Collection (Cambridge Butterfly)](https://huggingface.co/datasets/imageomics/Heliconius-Collection_Cambridge-Butterfly)
+
+394 dorsal images of _Heliconius erato_ and _Heliconius melpomene_ butterflies were sourced from this subset of Chris Jiggins' research group's collection from the University of Cambridge.
+
+![species distribution chart for CB data colored by label](https://github.com/user-attachments/assets/cee0dd24-00e8-46fd-93d0-02804b557fdf)
+
+
+
+### How to Access*
+
+Install the [cautious-robot](https://github.com/Imageomics/cautious-robot) package.
+
+Run
+```bash
+cautious-robot -i <path/to>/wing_seg_heliconius_collection.csv -o <path/to/output-directory> -s label -u source_url -l 256 -v "md5"
+```
+
+This will download all images into subfolders based on their label (`damaged`, `dorsal`, `body_attached`, or `incomplete`) and do the same for 256 x 256 downsized copies of the images (in a separate parent folder). It will then verify a match of all the full-sized image MD5s using the [sum-buddy package](https://github.com/Imageomics/sum-buddy).
+
+### Citation
+Full bibtex citations are provided in [Heliconius_collection_cambridge.bib](/heliconius_collection_cambridge.bib): these are for both the compilation and all original image sources from the Butterfly Genetics Group at University of Cambridge.
+
+## [Monteiro Data](https://lepdata.org/)
+
+Collection of 199 full body images of 105 species of butterflies across 16 genera from the Monteiro Lab (a subset of their collection).
+
+![Genus distribution chart for Monteiro data colored by label](https://github.com/user-attachments/assets/3075f07e-6166-48f8-8a31-53eec4d4a220)
+
+
+### How to Access*
+
+Install the [cautious-robot](https://github.com/Imageomics/cautious-robot) package.
+
+Run
+```bash
+cautious-robot -i <path/to>/wing_seg_monteiro.csv -o <path/to/output-directory> -s label -u source_url -l 256 
+```
+
+This will download all images into subfolders based on their label (in this case, just `body_attached`) and do the same for 256 x 256 downsized copies of the images (in a separate parent folder). 
+
+
+### Citation
+
+Images were accessed from the [Monteiro Lab website](https://lepdata.org/monteiro/).
+
+```
+@article{article,
+  author = {Silveira, Margarida and Monteiro, Antonia},
+  year = {2008},
+  month = {11},
+  pages = {130-6},
+  title = {Automatic recognition and measurement of butterfly eyespot patterns},
+  volume = {95},
+  journal = {Bio Systems},
+  doi = {10.1016/j.biosystems.2008.09.004}
+}
+```
+
+## [Smithsonian Tropical Research Institute (STRI) Data](https://huggingface.co/datasets/imageomics/STRI-Samples)
```

Comment:
Want to just mention `wing_seg_stri.csv` in this section to account for its presence in this repo?